4-perftools/test/run

```diff
@@ -44,6 +44,8 @@ function check_result() {
 function test_docker_run_usage () {
   info "Testing 'docker run' usage ..."
 
+  sed -i "s/RELEASE/$(docker run --rm $IMAGE_NAME uname -r)/" expected-usage
```

Comment:
Hm, I don't think it is a good idea to change sources of cloned git repo. I'd rather suggest to store the sed output to `$TMPDIR/expected-usage` and compare that file with `$TMPDIR/actual-usage` instead.